marketpulse/base/templates/base.html

```diff
@@ -74,7 +82,11 @@ <h2 class="hidden-xs">
     <script src="{{ static('js/libs/bootstrap.min.js') }}"></script>
     <script src="{{ static('js/site.js') }}"></script>
     <script src="//mozorg.cdn.mozilla.net/tabzilla/tabzilla.js"></script>
+    <script src="https://login.persona.org/include.js"></script>
+    <script src="{{ static('browserid/api.js') }}"></script>
+    <script src="{{ static('browserid/browserid.js') }}"></script>
```

Comment:
There is no need to include the two js libraries, you can use `{{ browserid_js }} instead after the js block.